panther_gazebo/config/led_strips.yaml

```diff
@@ -0,0 +1,11 @@
+chanel_1:
+  position: [0.362, 0.0, 0.201]
```

Comment:
Is there a simple way we could get this values from urdf?

Reply:
It probably is, but I don't know :/